.github/workflows/npm-audit.yml

```diff
@@ -23,3 +23,19 @@ jobs:
       - name: Run NPM Audit
         run: bash ${GITHUB_WORKSPACE}/hack/github-actions/npm_audit.sh
         shell: bash
+  npm-build-check:
+    name: npm-build-check
+    runs-on: ubuntu-latest
+    steps:
+      - name: setup Node.JS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.16.0
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      - name: Run NPM Build
+        working-directory: ./portal/v2
```

Comment:
Do we need this for both v1 and v2?

Reply:
We could add this for both v1 and v2. I just followed in the footsteps of the existing audit step that only runs against v2.

Reply:
I think I'm fine with just v2 since that's the direction we're heading in, yea? I imagine we will eventually deprecate v1 for the sake of maintainability?